sdk/python/conformance/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# Run conformance test and generate test report.
+pytest test/e2e/ --namespace=kf-conformance -v | tee /tmp/training-operator-conformance.log
+
+# Create the done file.
+touch /tmp/training-operator-conformance.done
+echo "Done..."
+
+# Keep the container running so the test logs can be downloaded.
+while true; do sleep 10000; done
```

Comment:
Should we copy logs to the local machine instead of keeping running Pod like KFP?

https://github.com/kubeflow/kubeflow/blob/2fa0d3665234125aeb8cebe8fe44f0a5a50791c5/conformance/1.5/Makefile#L37-L41

https://github.com/kubeflow/kubeflow/blob/2fa0d3665234125aeb8cebe8fe44f0a5a50791c5/conformance/1.5/report-pod.sh

Reply:
Can you please elaborate on this? How will we copy the logs? The container will not have access to the local machine, right?
Once the tests run successfully, we will have to use the `kubectl cp` command to copy the logs from outside the cluster.

Reply:
Sorry, this is my misunderstanding. Your implementation is right since this script will run on the Pod.